scripts/build_wasm.sh

```diff
@@ -46,18 +86,20 @@ command() {
     esac
 }
 
-case $1 in
-    "")
-        command "libs"
-        command "samples"
-        ;;
-    "libs")
-        command "libs"
-        ;;
-    "samples")
-        command "samples"
-        ;;
-    *)
-        echo "error: arg must be 'libs', 'samples', or empty to build both"
-        exit 1
-esac
+
+print_help() {
+    cat << END
+Usage: build_wasm.sh [OPTIONS] [TARGET]
+
+Options:
+  --help           Show help message.
+  --release        Enable release and size optimizations for the build.
+
+Targets:
+  libs             Specify to build libs.
+  samples          Specify to build samples.
+                   If omitted, both libraries and samples will be built.
```

Comment:
I'd revise https://github.com/hyperledger-iroha/iroha/tree/main/wasm#how-to-build to just direct to this help command

Reply:
Added.